### ISSUE TITLE
CLI should warn on nvme with default image

### ISSUE THREAD

**Issue by justinsb** [kubernetes/kops#3991]
We should try to warn users if they're using the default jessie image for their master, as it doesn't support the nvme mechanism we are using.

(It does support nvme, just the symlinks don't appear in /dev/disk/by-id, which is where we are looking for them)

**Comment by SharpEdgeMarshall** [comment]
So actually to use kops 1.8 and M5 instances what we should do?

**Comment by chrislovecnm** [comment]
Use the stretch image. kope.io/k8s-1.8-debian-stretch-amd64-hvm-ebs-2017-12-02 it is alpha in the channels.

**Comment by sybeck2k** [comment]
I'm having exactly the same problem. I'm using debian stretch but in fact /dev/disk/by-id is still not existing (there is /dev/disk/by-uuid/ which symlinks only the root disk).
Protokube then fails [here](https://github.com/kubernetes/kops/blob/master/protokube/pkg/protokube/aws_volume.go#L314). The etcd disks are correctly mounted as reported by fdisk:
```
Disk /dev/nvme1n1: 20 GiB, 21474836480 bytes, 41943040 sectors
...
Disk /dev/nvme2n1: 20 GiB, 21474836480 bytes, 41943040 sectors
...
```

I have to use a M5 instance as in the new Paris region is the only available (no M4). 
Any suggestions on how to workaround this? thanks!

**Comment by blakebarnett** [comment]
I think we should just warn about using m5/c5 at all.  So many problems have been reported with them...

**Comment by Typositoire** [comment]
Does it also affect `c5d` class ?

**Comment by geojaz** [comment]
closed in #5660
